docs/dictionary/command/save.lcdoc

```diff
@@ -23,21 +23,29 @@ save this stack as "Backup"
 Example:
 save stack "project1" as "/Disk/Folder/File"
 
+Example:
+save stack "MyOldApp" with format 7.0
```

Comment:
I think the valid formats should be listed here (7.0, 5.5, 2.7, 2.4)
It's not clear to me from the code what happens if an invalid format (e.g. 6.0) is requested.